### DISCUSSION TITLE
Any plans to support Google Analytics integration?

### DISCUSSION THREAD

**Discussion by kawpii** [lowdefy/lowdefy#1109] [Ideas and feature requests]
Google Analytics data is useful for dashboards any plans to support it as an integration?

**Comment by Gervwyk** [comment]
Hi @kaushalyap 
Yes, at the moment, you can already implement Google Analytics in v3 using the `appendHead` property. You can view [the example in the docs](https://docs.lowdefy.com/custom-code). Let me know if this is what you are looking for.
